<?xml version="1.0" encoding="UTF-8" ?>

<launch>
    <node pkg="localization" type="ICP_method1" name="icp" output="screen">
        <param name="max_xy" value="50" />
        <param name="max_car_x" value="30" />
        <param name="max_car_y" value="15" />
        <param name="max_car_z" value="10" />
        <param name="min_car_z" value="-10" />
    </node>
    <node pkg="localization" type="generate_csv.py" name="csv" output="screen"></node>
</launch>

<!-- Total score: 0.8927569240129892 0.6497990504534068 0.6758447818502624 0.432452223226028
final score: 0.7278520902941874
<param name="max_xy" value="60" />
<param name="max_car_x" value="40" />
<param name="max_car_y" value="15" />
<param name="max_car_z" value="10" />
<param name="min_car_z" value="-10" /> -->


<!-- Total score: 0.8884835311217799 0.736706399054576 0.6441083929670124 0.4071500267451876
final score: 0.7552018140417625
<param name="max_xy" value="50" />
<param name="max_car_x" value="30" />
<param name="max_car_y" value="15" />
<param name="max_car_z" value="10" />
<param name="min_car_z" value="-10" /> -->

